decidim-core/app/packs/src/decidim/index.js

```diff
@@ -16,10 +16,14 @@ import addInputEmoji, { EmojiButton } from "src/decidim/input_emoji"
 import dialogMode from "src/decidim/dialog_mode"
 import FocusGuard from "src/decidim/focus_guard"
 import backToListLink from "src/decidim/back_to_list"
+import markAsReadNotifications from "src/decidim/notifications"
+
+// NOTE: new libraries required to give functionality to redesigned views
 import Accordions from "a11y-accordion-component";
 import Dropdowns from "a11y-dropdown-component";
 import Dialogs from "a11y-dialog-component";
-import markAsReadNotifications from "src/decidim/notifications"
+import RemoteModal from "./redesigned_ajax_modals"
```

Comment:
Are we mixing redesign with the legacy layout or is this on purpose?

Reply:
On purpose, notice the "NOTE" wrapping these four lines. Some tests were broken, due to HTML changes